main.js

```diff
@@ -0,0 +1,123 @@
+"use script";
+var counters = {};
+var arr = [];
+var btnSort = document.getElementById('sort'); 
```

Comment:
Please, use apropriate names of the variables. Why 'btnSort' is not a button, but text input?